charts/athens-proxy/README.md

```diff
@@ -0,0 +1,133 @@
+# Athens Proxy Helm Chart
+
+## What is Athens?
+
+[Athens](https://docs.gomods.io) is a Server for Your Go Packages.
+
+Athens provides a server for [Go Modules](https://github.com/golang/go/wiki/Modules) that you can run. It serves public code and your private code for you, so you don't have to pull directly from a version control system (VCS) like GitHub or GitLab.
+
+## Prerequisites
+
+* Kubernetes 1.10+
+
+## Requirements
+
+- A running Kubernetes cluster
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed and setup to use the cluster
+- [Helm](https://helm.sh/) [installed](https://github.com/helm/helm#install) and setup to use the cluster (helm init) or [Tillerless Helm](https://github.com/rimusz/helm-tiller)
+
+## Deploy Athens
+
+The fastest way to install Athens using Helm is to deploy it from our public Helm chart repository. First, add the repository with this command:
+
+```console
+$ helm repo add gomods https://athens.blob.core.windows.net/charts
+$ helm repo update
+```
+
+Next, install the chart using no arguments.  
+
+```
+$ helm install gomods/athens-proxy -n athens --namespace athens
+```
+
+This will deploy a single Athens instance in the `athens` namespace with `disk` storage enabled. Additionally, a `ClusterIP` service will be created.
+
+## Advanced Configuration
+
+### Give Athens access to private repositories via Github Token (Optional)
```

Comment:
@rimusz this section all the way down to ".netrc file support" is also at https://docs.gomods.io/install/install-on-kubernetes/, so can you link to that here?

Also, it would be more helpful to move the last section about upstreams to those documentation instead of listing it here.